index.html

```diff
@@ -1,53 +1,182 @@
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-<link rel="stylesheet" href="style.css">
-    <title>technicalsupport</title>
-  </head>
-  <body>
-    <div class="header">
-      <div class="header-content">
-      <span class="pagetitle"></span><div class="logo"><div class="menubar">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="style.css">
+  <title>technicalsupport</title>
```

Comment:
Tittlul se pune de obicei laolalta cu meta tags si abia apoi incepi sa incarci resurse externe cum ar fi fisiere CSS sau JS. Motivul, pentru ca atunci cand browser-ul va trebui sa citeasca pagina html pentru a afisa-o, va trebui sa incarce fisierul css, caz in care, in functie de marimea acelui fisier, poate intarzia afisarea titlului. Elementele critice, daca se poate, trebuie sa fie puse cat mai sus in pagina, daca este posibil. Ai sa vezi cand ajungi sa incarci script-uri, ca unele trebuie sa le pui in footer sau sa adaugi anumite atribute care sa seteze felul cum vor fi incarcate acele script-uri